instrumentation/opentelemetry-instrumentation-kafka-python/src/opentelemetry/instrumentation/kafka/__init__.py

```diff
@@ -0,0 +1,102 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Instrument `kafka-python` to report instrumentation-kafka produced and consumed messages
+
+Usage
+-----
+
+..code:: python
+
+    from opentelemetry.instrumentation.kafka import KafkaInstrumentor
+    from kafka import KafkaProducer, KafkaConsumer
+
+    # Instrument kafka
+    KafkaInstrumentor().instrument()
+
+    # report a span of type producer with the default settings
+    producer = KafkaProducer(bootstrap_servers=['localhost:9092'])
+    producer.send('my-topic', b'raw_bytes')
+
+
+    # report a span of type consumer with the default settings
+    consumer = KafkaConsumer('my-topic',
+                             group_id='my-group',
+                             bootstrap_servers=['localhost:9092'])
+    for message in consumer:
+        # process message
+
```

Comment:
Could you include in the docs an example and description of using the `consume` and `produce` hooks? Similar to [this](https://github.com/open-telemetry/opentelemetry-python-contrib/blob/main/instrumentation/opentelemetry-instrumentation-elasticsearch/src/opentelemetry/instrumentation/elasticsearch/__init__.py#L46.